Documentation/manual-olm-install.md

```diff
@@ -117,14 +85,11 @@ Eventually your pod output should look like this:
 
 ```
 NAME                                  READY   STATUS    RESTARTS   AGE
-hdfs-datanode-0                       1/1     Running   0          7m24s
-hdfs-namenode-0                       1/1     Running   0          7m24s
-hive-metastore-0                      1/1     Running   0          7m24s
-hive-server-0                         1/1     Running   1          7m24s
-metering-operator-c7545d555-h5m6x     2/2     Running   0          8m44s
-metering-operators-bvpf7              1/1     Running   0          9m32s
-presto-coordinator-584789c6b-kpfpc    1/1     Running   0          7m24s
-reporting-operator-5c8db66985-9ghz4   1/1     Running   0          7m24s
+hive-metastore-0                      2/2     Running   0          3m28s
+hive-server-0                         3/3     Running   0          3m28s
+metering-operator-68dd64cfb6-2k7d9    2/2     Running   0          5m17s
+presto-coordinator-0                  2/2     Running   0          3m9s
+reporting-operator-5588964bf8-x2tkn   2/2     Running   0          2m40s
 ```
 
 Once all pods are ready, you can begin using Metering to collect and Report on your cluster.
```

Comment:
@chancez should we recommend that the user check `oc get reportdatasource` to make sure meterics have started to import before using Metering?

Reply:
Perhaps, let me think about it.

Reply:
Updated. What do you think @pruan-rht ?

Reply:
@chancez looks good, this is what the QE automated script does to validate the installation is successful before using.

Reply:
Cool thanks.